cpu/samd21/cpu.c

```diff
@@ -72,6 +94,22 @@ static void clk_init(void)
     while (!(SYSCTRL->PCLKSR.reg & SYSCTRL_PCLKSR_OSC8MRDY)) {}
 #endif
 
+    /* Setup GCLK2 with divider 1 (32.768kHz) */
+    GCLK->GENDIV.reg  = (GCLK_GENDIV_ID(2)  | GCLK_GENDIV_DIV(0));
+    GCLK->GENCTRL.reg = (GCLK_GENCTRL_ID(2) | GCLK_GENCTRL_GENEN
```

Comment:
We should either add `SAM0_GCLK_32KHZ` in the comment. Or else we can change the 2 in `SAM0_GCLK_32KHZ`.